edx2bigquery/main.py

```diff
@@ -2118,6 +2199,9 @@ def CommandLine():
         courses = get_course_ids(args)
         run_parallel_or_serial(logs2bq_single, param, courses, args, parallel=args.parallel)
 
+    elif (args.command=='logsfromS3'):
```

Comment:
why do you use parentheses ?

Reply:
I don't know, all the commands are like that.